### ISSUE TITLE
Drag does not work on Ipad / phone

### ISSUE THREAD

**Issue by ezequiel9** [cameronhimself/vue-drag-drop#79]
I can't make it work for ipad and phones. The drag is not working. Any suggestion?

**Comment by harsha935** [comment]
Yes, I have used this plugin in my project and now, I'm stuck on this

**Comment by yenntkaopiz** [comment]
Had anybody resolved it? :(( vue-drag-drop not working in Ipad & Iphone :((

**Comment by ezequiel9** [comment]
I don't think the problem is with this library. I have done a codepen to test the native html5 drag and drop feature and didn't work neither.  
Here the link:

https://codepen.io/ezequiel9/pen/gVxgBx

**Comment by harsha935** [comment]
Use these plugins. those are working on the tablet and mobile.
https://vuejsexamples.com/a-fast-and-lightweight-drag-drop-sortable-library-for-vue-js/
https://vuejsexamples.com/vue-component-for-draggable-and-resizable-elements/

**Comment by yenntkaopiz** [comment]
@ezequiel9  i tested your demo and it works well in ipad, problem from this library :(

**Comment by yenntkaopiz** [comment]
@harsha935  thank u, try it now

**Comment by yenntkaopiz** [comment]
@cameronhimself  do you have any suggest for this case? thank you. i really like your library
